src/OpenApi/test/Microsoft.AspNetCore.OpenApi.Tests/Services/CreateSchemaReferenceIdTests.cs

```diff
@@ -199,7 +200,7 @@ await VerifyOpenApiDocument(builder, options, document =>
         });
     }
 
-    [Fact]
+    [ConditionalFact(Skip = "https://github.com/dotnet/aspnetcore/issues/58619")]
     public async Task HandlesDuplicateSchemaReferenceIdsGeneratedByOverload()
```

Comment:
This is one of the tests we have to temporarily skip while we sort out how schema deduping looks in the new model.